src/runmode-unix-socket.c

```diff
@@ -324,7 +316,7 @@ static TmEcode UnixSocketAddPcapFileImpl(json_t *cmd, json_t* answer, void *data
     PcapCommand *this = (PcapCommand *) data;
     const char *filename;
     const char *output_dir;
-    int tenant_id = 0;
+    uint32_t tenant_id = 0;
```

Comment:
this is initialized below with `tenant_id = json_number_value(targ);`. `json_number_value()` returns a `double`. I think it would be good to add some bounds checking before casting to `uint32_t`

Reply:
Sorry about not getting back and following through on this work. I'll continue on this now, look into adding a bounds check and update the PR. Thanks

Reply:
This is what we are doing in that block of code:

```
    json_t *targ = json_object_get(cmd, "tenant");
    if (targ != NULL) {
        if (!json_is_integer(targ)) {
            json_object_set_new(answer, "message",
                                json_string("tenant is not a number"));
            return TM_ECODE_FAILED;
        }
        tenant_id = json_number_value(targ);
    }
```

Since we are doing a `json_is_integer` check (and not a `json_is_real` check), is it safe to call `json_integer_value` instead of the current `json_number_value` @victorjulien? Thanks